src/services/activation/utils/customUtils.js

```diff
@@ -0,0 +1,66 @@
+// ***************** IMPORTANT *****************
```

Comment:
why is this file called customUtils? are the utils in index.js not custom? ;P

Maybe call this file quarantinedObject.js or something

Reply:
or something that makes clear that these are supposed to be mongoose hooks.

Reply:
Yes, they are custom :P, but I wanted to emphasize that changes should be made here when a new service is written, for example: add keyword. The other utils are available for all services and are compareable to basic building blocks